src/parser/jobs/blu/modules/DoTs.tsx

```diff
@@ -72,36 +100,92 @@ export class DoTs extends CoreDoTs {
+			</Trans>,
+			tiers: {1: SEVERITY.MAJOR},
+			why: <Trans id ="blu.breath_of_magic.suggestion.unbristled.why">
+				<Plural value={this.unbuffedBoM} one="# Breath of Magic was" other="# Breaths of Magic were"/> cast without both buffs.
+			</Trans>,
+			value: this.unbuffedBoM,
+		}))
+
+		this.suggestions.add(new TieredSuggestion({
+			icon: this.data.actions.MORTAL_FLAME.icon,
+			content: <Trans id = "blu.mortal_flame.suggestion.unbristled.content">
+				<DataLink action="MORTAL_FLAME"/> must always be buffed with <DataLink action="BRISTLE"/> and <DataLink action="MOON_FLUTE"/> for a colossal damage gain.
+			</Trans>,
+			tiers: {1: SEVERITY.MAJOR},
+			why: <Trans id ="blu.mortal_flame.suggestion.unbristled.why">
+				<Plural value={this.unbuffedMortal} one="# Mortal Flame was" other="# Mortal Flames were"/> cast without both buffs.
+			</Trans>,
+			value: this.unbuffedMortal,
+		}))
 	}
 
 	protected override addChecklistRules() {
-		const uptimePercent = this.getUptimePercent(this.data.statuses.BLEEDING.id)
+		const SoTUptimePercent = this.getUptimePercent(this.data.statuses.BLEEDING.id)
+		const requirements: Requirement[] = [
+			new Requirement({
+				name: <Trans id="blu.dots.requirement.uptime-bleed.name"><DataLink status="BLEEDING" showIcon={false} /> uptime</Trans>,
+				percent: SoTUptimePercent,
+			}),
+		]
+
+		const bomUptimePercent = this.getUptimePercent(this.data.statuses.BREATH_OF_MAGIC.id)
+		if (bomUptimePercent > 1) {
+			requirements.push(new Requirement({
+				name: <Trans id="blu.dots.requirement.uptime-bom.name"><DataLink status="BREATH_OF_MAGIC" showIcon={false} /> uptime</Trans>,
+				percent: bomUptimePercent,
+			}))
+		}
+
+		const flameUptimePercent = this.getUptimePercent(this.data.statuses.MORTAL_FLAME.id)
+		if (flameUptimePercent > 1) {
+			requirements.push(new Requirement({
+				name: <Trans id="blu.dots.requirement.uptime-flame.name"><DataLink status="MORTAL_FLAME" showIcon={false} /> uptime</Trans>,
+				percent: flameUptimePercent,
+			}))
+		}
+
 		this.checklist.add(new TieredRule({
 			name: <Trans id="blu.dots.rule.name">Keep your DoTs up </Trans>,
 			description: <Trans id="blu.dots.rule.description">
-				The <DataLink status="BLEEDING" showIcon={false} showTooltip={false} /> effect from Nightbloom and SoT is a solid 15% of your total damage.
+				The <DataLink status="BLEEDING" showIcon={false} showTooltip={false} /> effect from <DataLink action="NIGHTBLOOM" showIcon={false} showTooltip={false} /> and <DataLink action="SONG_OF_TORMENT" showIcon={false} showTooltip={false} /> is a solid 15% of your total damage. <br />
+			If you are one of the DPSes applying <DataLink action="BREATH_OF_MAGIC" showIcon={false} showTooltip={false} /> or <DataLink action="MORTAL_FLAME" showIcon={false} showTooltip={false} />, it should be a top priority to keep these DoTs rolling.
```

Comment:
Something I missed in earlier review passes: We're trying to discourage the use of `TieredRule` since it muddies the intended meaning of the checklist as a simple 'did you or did you not do `thing`' check. Can you change this to a regular `Rule` and skip the 90% warning target? The 95% success target is sufficient.